packages/shared/src/components/cards/v1/ArticlePostCard.tsx

```diff
@@ -94,26 +104,39 @@ export const ArticlePostCard = forwardRef(function PostCard(
             >
               <SourceButton size="large" source={post.source} />
             </PostCardHeader>
-            <CardTitle
-              lineClamp={undefined}
-              className={!!post.read && 'text-theme-label-tertiary'}
-            >
-              {post.title}
-            </CardTitle>
-          </CardTextContainer>
-          {post.summary && (
-            <CardTextContainer className="mt-4 text-theme-label-secondary">
-              {post.summary}
-            </CardTextContainer>
-          )}
+
+            <CardContent>
+              <div className="mr-4 flex-shrink flex-grow">
+                <CardTitle
+                  lineClamp={undefined}
+                  className={!!post.read && 'text-theme-label-tertiary'}
+                >
+                  {post.title}
+                </CardTitle>
+
+                {post.summary && (
+                  <CardTextContainer className="mt-4 text-theme-label-secondary">
+                    {post.summary}
+                  </CardTextContainer>
+                )}
+              </div>
```

Comment:
Are the title and summary limits set based on the notes on DR? https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/681410561/WT-1957+Feed+Layout+Improvements#Main

Ex of one of the notes for limits:
All posts (Title + Tldr) will be limited to 300 chars total

Reply:
Done, thank you for pointing it out